peachjam/js/components/DocumentProblemModal.vue

```diff
@@ -83,7 +120,18 @@ export default {
       problem: '',
       submitted: false,
       success: true,
-      url: window.location.toString()
+      url: window.location.toString(),
+      selectedOption: '',
+      other: '',
+      isVisible: true,
+      options: [
```

Comment:
I think it's easier simply to put the options directly in the HTML inside the select element.

Additionally, the labels (not the values) must be translateable.